recipes/pymispwarninglists/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pymispwarninglists" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c60eb6fecf415a4e122af92c4c22ab14553a52fb02c8e6e0809ce1e8adc05a47"
+
+build:
```

Comment:
I believe this package can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python